### DISCUSSION TITLE
Question: What does the "Version bump only for package @standardnotes/mobile" changelog mean in github releases?

### DISCUSSION THREAD

**Discussion by Universal-Igloo** [standardnotes/forum#2878] [General]
What does the "Version bump only for package @standardnotes/mobile" changelog mean in github releases?

Sometimes there are 4 updates a day, each with the same changelog and each around 85 MB in size, is there a difference between 2 updates with that same changelog on the same day or is just a version number bump (with no code changes) that we are downloading for 85 MBs?

Just wanted to clarify 😄

**Comment by virtuallyvlad** [comment]
Are you talking about the pre-release updates?  I wouldn't download those; and you can hide them from that page by [searching `prerelease:false`](https://github.com/standardnotes/app/releases?q=prerelease%3Afalse).  But yea, I am curious too about why the version bumps.
